docs/blocks-storage/store-gateway.md

```diff
@@ -698,6 +698,15 @@ blocks_storage:
     # CLI flag: -blocks-storage.tsdb.flush-blocks-on-shutdown
     [flush_blocks_on_shutdown: <boolean> | default = false]
 
+    # If TSDB has not received any data for this duration, and all blocks from
+    # TSDB have been shipped, TSDB is closed and deleted from local disk. If set
+    # to positive value, this value should be equal or higher than
+    # -querier.query-ingesters-within flag to make sure that TSDB is not closed
```

Comment:
Does this comment apply on the store-gateway as well, on being greater than query-ingesters-within?

Reply:
No, it doesn’t, but store-gateways share the same config struct, which is why it shows up in this documentation as well. We need to fix that.

Reply:
Store-gateways don’t have TSDB open at all.